wordpress-mobile/WordPress-iOS
# Issue/6072 improve notifications sync after quick actions

**Fixes** #6072 

**To test:**
Trigger each of the possible quick actions in a Comment Notification: Reply, Like and Approve.

After you've taken a quick action check that the following happened:
1. After a few seconds the application badge number will be updated to account for the action taken.
2. If you navigate to the Notifications screen you will see that for all the notifications that you have taken an action for, they are effectively marked as read.
3. The Notification action buttons and state reflect the action taken.

Needs review: @jleandroperez
